src/db/OrganizationSchema.ts

```diff
@@ -0,0 +1,69 @@
+import mongoose from 'mongoose'
+import muuid from 'uuid-mongodb'
+
+import { OrganizationType, OrgType, IOrganizationContent, OperationType } from './OrganizationTypes.js'
+import { ChangeRecordMetadataType } from './ChangeLogType.js'
+
+const { Schema, connection } = mongoose
+
+const ChangeRecordMetadata = new Schema<ChangeRecordMetadataType>({
+  user: {
+    type: 'object',
+    value: { type: 'Buffer' },
+    required: true
+  },
+  historyId: { type: Schema.Types.ObjectId, ref: 'change_logs' },
+  operation: {
+    type: Schema.Types.Mixed,
+    enum: Object.values(OperationType),
+    required: true
+  },
+  seq: { type: Number, required: true, default: 0 }
+}, { _id: false, timestamps: false })
+
+const ContentSchema = new Schema<IOrganizationContent>({
+  website: { type: Schema.Types.String },
+  email: { type: Schema.Types.String },
+  donationLink: { type: Schema.Types.String },
+  instagramLink: { type: Schema.Types.String },
+  description: { type: Schema.Types.String }
+}, { _id: false })
+
+export const OrganizationSchema = new Schema<OrganizationType>({
+  orgId: {
+    type: 'object',
+    value: { type: 'Buffer' },
+    default: () => muuid.v4(),
+    required: true,
+    unique: true,
+    index: true
+  },
+  displayName: { type: String, required: true, index: true },
+  orgType: {
+    type: Schema.Types.Mixed,
+    enum: Object.values(OrgType),
+    required: true
+  },
+  associatedAreaIds: [{ type: 'Buffer' }],
```

Comment:
Decided to use the `area_id`s which are MUUIDs here, instead of `area._id` which are Mongo `ObjectId`s. After all, both are indexed, so should be roughly as fast as each other (Maybe Mongo has special optimizations for their own native _ids?) But the big upside is that `area_id`s are externally visible, so we can pass them in directly and filter directly rather than having to pass them in and translate into `area._ids`.